src/Notification/__stories__/Notifications.stories.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, boolean, select, number } from '@storybook/addon-knobs';
+
+import { NotificationProvider, useNotifications } from '../provider';
+import Button from '../../Button';
+import Message from '../../Message';
+import NotificationReadme from '../README.md';
+
+const NotifiationComponent = ({ autoDismiss, autoDismissTimeout, pauseOnHover }) => {
+  const { addNotification, dismissNotification } = useNotifications();
+  const Component = () => {
+    return <Message description="this is a message" type="success" onClose={dismissNotification} />;
+  };
```

Comment:
I don't know if this is relevant but you can make your Component more readable by making an implicit value return :
```js
const Component = () => <Message description="this is a message" type="success" onClose={dismissNotification} />;
```